src/main/java/seedu/address/model/person/Person.java

```diff
@@ -23,11 +23,12 @@ public class Person {
 
     // Data fields
     private final Address address;
-
     private final Set<Tag> tags = new HashSet<>();
+    private boolean isPinned;
```

Comment:
Would implementing and using a PinStatus class instead of a boolean primitive allow further modularity and possible extension to the feature in the future? As an added bonus, this would also allow uniformity with the rest of the Person's details which have their own types!